docker/Dockerfile

```diff
@@ -1,17 +1,20 @@
-FROM mcr.microsoft.com/dotnet/aspnet:5.0-alpine
+FROM mcr.microsoft.com/dotnet/sdk:5.0-alpine as builder
 
-COPY ./compiler /RESTler/compiler
-COPY ./restler /RESTler/restler
+RUN apk add --no-cache python3 py3-pip
+RUN ln -s /usr/bin/python3 /usr/bin/python
 
-#Get the source code for the engine, and
-#copy source code to the image
-#RESTler Python dependency
-RUN apk add --no-cache python3
-RUN python3 -m ensurepip
-RUN pip3 install --upgrade pip
-RUN pip3 install requests
-RUN pip3 install requests
-RUN pip3 install applicationinsights
-COPY ./engine /RESTler/engine
-RUN python3 -m compileall -b /RESTler/engine
-COPY ./resultsAnalyzer /RESTler/resultsAnalyzer
+COPY src ./src
+COPY restler ./restler
```

Comment:
This indeed does not work with the existing pipeline, the context needs to be updated (I will point you to a version to use).

Reply:
The context is the root of this repo. So the build command would be `docker build -f docker/Dockerfile .` I've added this to the README.

It would probably be even more clear if the `Dockerfile` was moved to the root of the repo. Would you agree?

Reply:
Hello,

My apologies, just getting back to this now.  Yes, definitely, it would be more clear if the Dockerfile is at the root.  

How about we do two updates, so we do not need to coordinate the CI changes?

1) For this PR, add the Dockerfile with your latest changes to the root, and leave the docker/Dockerfile as is.  

2) I will follow up with a change to get CI working using the new dockerfile, and remove the old one.

Reply:
Yes, agreed. I've changed the files as you've described in step 1.